src/styles/foundation/_focus-ring.scss

```diff
@@ -20,27 +20,19 @@
       bottom: calc(-1 * #{$position});
       left: calc(-1 * #{$position});
       display: block;
-      border: $stroke solid var(--p-focused);
-      transition: opacity duration(fast) var(--p-ease),
-        transform duration(fast) var(--p-ease);
-      opacity: 0;
-      transform: scale(0.8);
+      box-shadow: 0 0 0 -#{$offset} var(--p-focused);
```

Comment:
The offset would vary because of the `::after` selector. Without a border on the element, it would be 1px, but if the element has a 2px border lets say, for example checkbox, the offset will be `2px + 1px`

Reply:
We should look at the performance here as well because animating the scale is better performant than the box-shadow. The shadow will cause a re-paints but the scaling and opacity will not.